cfgov/v1/atomic_elements/organisms.py

```diff
@@ -264,31 +264,58 @@ def make_created_value(self, instance, value):
     fields = None
     field_headers = None
 
-    row_links = blocks.BooleanBlock(
+    first_row_is_table_header = blocks.BooleanBlock(
```

Comment:
@richaagarwal I agree this is somewhat suboptimal. The way that `ModelTable` works is that it kind of wraps a Wagtail table block (formerly our `Table`, with this PR our `AtomicTable`). It would be nice to be able to use inheritance to avoid this, and I would like to revisit in future to see if this is possible.

For now, what these elements do is add the various `AtomicTable` options to the `ModelTable`:

![image](https://cloud.githubusercontent.com/assets/654645/18142019/b6d80f8a-6f89-11e6-93a1-adf146753d22.png)

It'd be better not to have this duplication, but without adding these options there'd be no way to set them on the table.